src/revChatGPT/V3.py

```diff
@@ -57,6 +57,8 @@ def __init__(
             ],
         }
         self.system_prompt = system_prompt
+        if max_tokens > 4096:
+            raise Exception("Max tokens cannot be greater than 4096")
```

Comment:
Limit should be 4000 because of some tokens used for formatting we have no control over